src/main/java/de/dentrassi/rpm/builder/VerifyDetails.java

```diff
@@ -0,0 +1,159 @@
+package de.dentrassi.rpm.builder;
+
+import org.eclipse.packager.rpm.VerifyFlags;
+import org.eclipse.packager.rpm.build.FileInformation;
+
+import java.util.EnumSet;
+import java.util.Set;
+
+/**
+ * Each member of this class corresponds to one bit in the verify flags.
+ * See http://ftp.rpm.org/api/4.14.0/group__rpmvf.html.
+ * See https://github.com/ctron/rpm-builder/issues/41
+ */
+public final class VerifyDetails {
+   private Boolean md5;
```

Comment:
Wouldn't it be easier to use a `boolean` here, and default to false?

Reply:
I agree. I copied the pattern from de.dentrassi.rpm.builder.EntryDetails. I assumed that there was some reason for that pattern?!
That pattern does have and issue though, see #42.
I am fine with changing the types in VerifyDetails it to primitive type boolean if you say so, but then what about EntryDetails?

Reply:
I have just changed to the type to boolean and fixed silly javadoc warnings.

Reply:
Hm … interesting. The idea of EntryDetails was that it would override the current entry it processes. So if the value set, it will be set. If the value is "null" it will not change it.

Your original code is close to that, and now I understand what you had in mind.

I think it good to keep the `Boolean`, but change the "transfer" steps into a way, that they do process on the existing set. If I understand your code correctly, then this would provide the same behavior. If the entry is set, it will be applied (set or unset), if the value is "null", it won't change the flag.

Reply:
As far as the boolean flags in a bitmap are concerned, we do not have the option of "not changing" something. Either is is set or it is not. So the three values (null, true, false) will be mapped to two values (0,1). My cristisim for #42 is that for EntryDetails.readmy etc, that mapping is counterintuitive.

In this pull-request, there is one bitmap (org.eclipse.packager.rpm.build.RpmBuilder.FileEntry#verifyFlags).
In the old implementation, its value had the hard-coded value -1, so every flag was set. 
My proposed change (or, rather, your proposal the way I understood it) is that the programmer has two options:
1. He/she does not specify anything about verify-flags, then the behaviour is unchanged, all flags will be set and everthing will be verified; or
2. He/she specifies a (possibly empty) list of those flags that shall be verified, then only those will be set in the bitmap and only the corresponding aspects will be verified.

Now it is not clear what exactly is meant by "if the value is 'null', it won't change the flag".

I am open to completely reconsider the question of how the programmer shall specify the verify-flags. It is a considerable idea to let the programmer specify a negative-list rather than a positive-list. This way, the default for each flag would be 'true' (or 1, respectively), and the user would specify the flags that shall be *excluded* from the bitmap.
However, while this might seem logical from the programmer's perspective, I consider it non-intuitive for the programmer. 

BTW, I think that only very few combinations of these flags are relevant for practical purposes. If the user had to specify all flags to exclude if he/she wishes to exclude any, then this would IMHO severly complicate the usage. 
My primary use-case (which is, in fact, the only one I can think of) is to customize the specification for configuration files. For such a file, the administrator of the target system may edit the file, which will change size, mtime, and digest at the same time.

So yet another approach to let the programmer specify the verify-flags would be to let him/her specify one of two or three bitmasks that actually make sense. 

In conclusion: the current proposal (implemented with boolean rather than Boolean) is a reasonable approach, and for the sake of simplicitly I would favour to keep it that way.

Reply:
Hm, I see your point … so, you consider the set of flags an entity by itself.

For the entry details, the idea was that every step would have the possibility to contribute to the final file entry. So you could have a rule, which matches e.g. `/etc/` and flags all of those files as "configuration". But doesn't set any other attributes of the file.

Now I could imagine that someone might want to do a similar thing with the verification flags. Then again, no one asked for that :grinning: 

So just let me know if your are fine with the current state, and I will merge it.

Reply:
I am satisfied with the current state and woul appreciate your merging.